Library/Homebrew/extend/ARGV.rb

```diff
@@ -13,7 +13,7 @@ def flags_only
 
   def formulae
     require "formula"
-    @formulae ||= (downcased_unique_named - casks).map { |name| Formulary.factory(name, spec) }
+    @formulae ||= (downcased_unique_named - casks).map { |name| Formulary.factory_with_priority(name, spec) }
```

Comment:
Oh I see. It will be much clearer in this way.

However I suppose we should still prioritize formulae paths like `ack.rb` or bottle paths? So probably in `ARGV#formulae` we need to test similar to the first few lines in `Formulary#loader_for`?